### PR TITLE
Reference Artifact Updates

### PR DESCRIPTION
**Pull request by CooperStansbury**
Please be thorough during review. Will squash when the time comes. 

1. Updated ontology base url.
1. Added classes per working version of the `spek`. 
1. Updated Readme.

### CODE REVIEW

File: slowmo.owl [hunk 1]
```diff
@@ -127,240 +219,669 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
...
+
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Annotations
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    <rdf:Description rdf:about="http://purl.org/dc/elements/1.1/">
```
Comment (grosscol) [inline slowmo.owl]: The entire following section seems peculiar to me.  It looks like an artifact of protege merging in parts of DC11.  Adding definitions for DC11 classes strikes me as redundant.  These should all have already been defined in `http://purl.org/dc/elements/1.1/` and that should have been pulled in by a namespace statement.

I wonder if the `xmlns:terms="http://purl.org/dc/terms/"` results in protege grabbing the dc11 terms in this fashion.

In any event.  This is not a blocker.  Having these here is find for the time being.

File: slowmo.owl [hunk 2]
```diff
@@ -80,38 +122,88 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
...
-    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/RO_0002434"/>
+    <owl:ObjectProperty rdf:about="http://example.com/slowmo#IsAboutISR">
+        <rdfs:subPropertyOf rdf:resource="http://example.com/slowmo#IsAbout"/>
+        <rdfs:label>is about ISR</rdfs:label>
+    </owl:ObjectProperty>
     
 
 
-    <!-- http://purl.obolibrary.org/obo/slowmo#slowmo_0000001 -->
+    <!-- http://example.com/slowmo#IsAboutPerformer -->
```
Comment (grosscol) [inline slowmo.owl]: This is a much nicer IRI to deal with internally.  I concur with the decision to make the IRIs human readable camel case.

File: slowmo.owl (hunk 1 above)
Comment (CooperStansbury) [reply slowmo.owl]: @grosscol took a look and I think these we coming from an unneeded import. Removed for now (f756512 ) and removing the `catalog-v001.xml` files from `.gitignore`.

**Comment by CooperStansbury** [comment]
@grosscol squashed, merge at will

**Pull request merged**